src/lib.rs

```diff
@@ -1,10 +1,48 @@
+//! Epoch-based memory reclamation.
+//!
+//! # Pointers
+//!
+//! Concurrent collections are built using atomic pointers. This module provides [`Atomic`], which
+//! is just a shared atomic pointer to a heap-allocated object. Loading an [`Atomic`] yields a
+//! [`Ptr`], which is an epoch-protected pointer through which the loaded object can be safely read.
+//!
+//! # Pinning
+//!
+//! Before an [`Atomic`] can be loaded, the current mutator must be [`pin`]ned. By pinning a mutator
```

Comment:
@jeehoonkang Could you explain a bit what is the motivation behind *realms*, and in particular why would someone want multiple *realms*?